src/test/scala/uk/gov/nationalarchives/tdr/api/utils/TestUtils.scala

```diff
@@ -197,5 +201,59 @@ object TestUtils {
 
     ps.executeUpdate()
   }
+
+  def addConsignmentProperty(name: String): Unit = {
+    // name is primary key check exists before attempting insert to table
+    if (!propertyExists(name)) {
+      val sql = s"insert into ConsignmentProperty (Name) VALUES (?)"
+      val ps: PreparedStatement = DbConnection.db.source.createConnection().prepareStatement(sql)
+      ps.setString(1, name)
+      ps.executeUpdate()
+    }
+  }
+
+  private def propertyExists(name: String): Boolean = {
+    val sql = s"SELECT * FROM ConsignmentProperty WHERE Name = ?"
+    val ps: PreparedStatement = DbConnection.db.source.createConnection().prepareStatement(sql)
+    ps.setString(1, name)
+    val rs: ResultSet = ps.executeQuery()
+    rs.next()
+  }
+
+  //scalastyle:off magic.number
+  def addConsignmentMetadata(metadataId: String, consignmentId: String, propertyName: String): Unit = {
+    val sql = s"insert into ConsignmentMetadata (MetadataId, ConsignmentId, PropertyName, Value, Datetime, UserId) VALUES (?, ?, ?, ?, ?, ?)"
+    val ps: PreparedStatement = DbConnection.db.source.createConnection().prepareStatement(sql)
+    ps.setString(1, metadataId)
+    ps.setString(2, consignmentId)
+    ps.setString(3, propertyName)
+    ps.setString(4, "Result of ConsignmentMetadata processing")
+    ps.setTimestamp(5, Timestamp.from(FixedTimeSource.now))
+    ps.setString(6, userId.toString)
+
+    ps.executeUpdate()
+  }
+
+  def createConsignmentMetadata(consignmentId: UUID): Unit = {
```

Comment:
Should this be called `addTransferAgreementMetadata`? Otherwise it's hard to tell from the method name that it does transfer agreement stuff, and it also looks confusing similar to `addConsignmentMetadata`.

Reply:
Fixed